File: test/aidbox_sdk/snapshots/dotnet/hl7-fhir-r4-core/OrganizationAffiliation.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class OrganizationAffiliation : DomainResource, IResource
{
    public Base.CodeableConcept[]? Specialty { get; set; }
    public Base.ResourceReference? Organization { get; set; }
    public Base.ResourceReference? ParticipatingOrganization { get; set; }
    public bool? Active { get; set; }
    public Base.CodeableConcept[]? Code { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.ContactPoint[]? Telecom { get; set; }
    public Base.ResourceReference[]? Network { get; set; }
    public Base.Period? Period { get; set; }
    public Base.ResourceReference[]? Location { get; set; }
    public Base.ResourceReference[]? Endpoint { get; set; }
    public Base.ResourceReference[]? HealthcareService { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/OrganizationAffiliation.py
from typing import Optional, List
from pydantic import *
from ..base.Period import Period
from ..base.CodeableConcept import CodeableConcept
from ..base.ContactPoint import ContactPoint
from ..base.DomainResource import DomainResource
from ..base.Reference import Reference
from ..base.Identifier import Identifier

class OrganizationAffiliation(DomainResource):
    specialty: Optional[List[CodeableConcept]] = None
    organization: Optional[Reference] = None
    participating_organization: Optional[Reference] = None
    active: Optional[bool] = None
    code: Optional[List[CodeableConcept]] = None
    identifier: Optional[List[Identifier]] = None
    telecom: Optional[List[ContactPoint]] = None
    network: Optional[List[Reference]] = None
    period: Optional[Period] = None
    location: Optional[List[Reference]] = None
    endpoint: Optional[List[Reference]] = None
    healthcare_service: Optional[List[Reference]] = None